src/docs/resources/faq.md

```diff
@@ -425,7 +425,12 @@ adaptations in our widgets.
 
 ### Does Flutter run on the web?
 
-No. We are not working to provide a web version of Flutter.
+Yes. We're working on an experimental project called [Hummingbird][]. It is a
```

Comment:
I would not say yes, but maybe "We are working on..."

Reply:
Thanks, I've dropped the "yes".